docs/source/Components/index.rst

```diff
@@ -64,36 +65,38 @@ asset output is represented as a sequence of textured meshes
+      for all applications that require streaming and playback that is longer than a few seconds.
+      More info about an MPD structure can be found in the `MPD structure <Mpd.html>`_ document
 
-    glTF - This output can be used in standard editors/players and social platforms. glTF is widely adopted format, particularly suitable for the web.
-    However, this format performs poorly for longer content where progressive streaming is required.
+    * glTF - This output can be used in standard editors/players and social platforms. glTF is widely adopted format, particularly suitable for the web.
+      However, this format performs poorly for longer content where progressive streaming is required.
 
-    FBX - This output is primarily used for cases where the output needs to be edited using
-    tools like Maya. After editing, this format can be converted to any of the other formats.
+    * FBX - This output is primarily used for cases where the output needs to be edited using
+      tools like Maya. After editing, this format can be converted to any of the other formats.
 
 **Difference from HVR**
 
 The main difference between HVR and PAT is how the underlying 3D asset is represented.
 HVR takes the approach of representing the 3D asset as a point cloud. However, PAT represents the asset as a textured mesh.
 Effectively, PAT decouples the mesh(geometry) resolution from the texture(color) resolution. Allowing us to scale up the asset perceived resolution, with smaller increase in overall asset size.
-To illustrate it, think of an actor with a very textured shirt, the underlying geometry of the shirt is fairly simple, allowing us to save a lot of geometry bandwidth. In this case most the complexity is on the texture side, which we can very effectively encode using standard video codecs.
+To illustrate it, think of an actor with a very textured shirt. The underlying geometry of the shirt is fairly simple, allowing us to save a lot of geometry bandwidth. In this case most the complexity is on the texture side, which we can very effectively encode using standard video codecs.
 The represent the high details with HVR, we will need a very high resolution of points, causing the file size to explode.
 
-**Unity usage with Unity**
+**PAT usage with Unity**
 
-HVR is using a point cloud to reprensent the data. Point clouds are less standatd and hence Unity doesnt do as good job rendering them.
+HVR is using a point cloud to represent the data. Point clouds are less standard and hence Unity's standard rendering will not produce the best output to the user.
 As a result, to produce the best quality render, the HVR plugin takes over the rendering of the asset. This internal rendering takes away some of feature set that a developer would expect with a 3D asset in Unity.
 In contrast, PAT simply delivers a mesh and a texture per frame, giving the developer back the control over rendering and 3D asset manipulation and effects.
```

Comment:
Maybe "giving back the control over rendering and 3D asset manipulation and effects to the developer."